Examples/Scripts/Historical/7T/GenericfMRIVolumeProcessingPipelineBatch.7T.sh

```diff
@@ -208,11 +208,20 @@ do
 		
 		# Not using Siemens Gradient Echo Field Maps for susceptibility distortion correction
 		# Set following to NONE if using TOPUP
-		MagnitudeInputName="NONE" #Expects 4D Magnitude volume with two 3D volumes (differing echo times)
-		PhaseInputName="NONE" #Expects a 3D Phase difference volume (Siemen's style)
-		DeltaTE="NONE" #2.46ms for 3T, 1.02ms for 7T
-		
-		# Not using General Electric Gradient Echo Field Maps for Distortion Correction
+		# or set the following inputs if using regular FIELDMAP (i.e. SiemensFieldMap GEHealthCareFieldMap PhilipsFieldMap)
```

Comment:
Implement same changes to the ".7T.sh" script that I suggested for the "3T" version.